patches/runtime/0017-Skip-some-test-projects.patch

```diff
@@ -0,0 +1,37 @@
+From 16cd6f6c480f50e5312815cf0a01cde0bee2d019 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Wed, 9 Sep 2020 09:55:42 -0500
+Subject: [PATCH 17/17] Skip some test projects
+
+---
+ eng/Subsets.props | 6 ++----
+ 1 file changed, 2 insertions(+), 4 deletions(-)
+
+diff --git a/eng/Subsets.props b/eng/Subsets.props
+index fa72f142f0c..17103c8c611 100644
+--- a/eng/Subsets.props
++++ b/eng/Subsets.props
+@@ -59,7 +59,7 @@
+     <DefaultMonoSubsets Condition="'$(MonoEnableLLVM)' == 'true' and '$(MonoLLVMDir)' == ''">mono.llvm+</DefaultMonoSubsets>
+     <DefaultMonoSubsets>$(DefaultMonoSubsets)mono.runtime+mono.corelib</DefaultMonoSubsets>
+ 
+-    <DefaultLibrariesSubsets>libs.depprojs+libs.native+libs.ref+libs.src+libs.pretest+libs.packages</DefaultLibrariesSubsets>
```

Comment:
How does this impact the source-build build?  Doesn't source-build specify it's own subsets in the patched int sb-specific build script?  Are the defaults used?

Reply:
No, for libraries the default (`libs`) is used (see https://github.com/dotnet/source-build/blob/092d4d5b76a1365add295de3630d5e27ee13f027/patches/runtime/0003-Add-source-build-specific-build-script.patch#L99).